jekyll/_cci2/machine-runner-3-configuration-reference.adoc

```diff
@@ -233,6 +233,25 @@ runner:
   mode: continuous
 ```
 
+When using `single-task` mode, the expected behaviour is that the runner process will while the machine instance will continue. However you may find it useful to terminate the machine instance itself.
```

Comment:
Is there a word missing? "...will while..."

Reply:
thanks for the catch